app/models/Capacity.scala

```diff
@@ -19,14 +19,30 @@ package models
 import java.time.LocalDate
 
 import play.api.libs.json.Json
-
+import exceptions._
 case class Capacity(capacity: CapacityType, fromDate: LocalDate, toDate: Option[LocalDate])
 
 object Capacity {
   implicit val format = Json.format[Capacity]
 
   lazy val defaultFromDate = LocalDate.of(2017, 4, 1)
 
-  def fromDeclaration(declaration: CapacityDeclaration) =
-    Capacity(declaration.capacity, declaration.fromDate.getOrElse(defaultFromDate), declaration.toDate)
+  def apply(linkingSession: LinkingSession): Capacity =
+    new Capacity(
+      linkingSession.propertyRelationship
+        .map { relationship =>
+          relationship.capacity
+        }
+        .getOrElse(
+          throw new ClaimPropertyRelationshipIsEmptyException("property claim relationship should not be empty")),
```

Comment:
PropertyRelationshipException is shorter :) The exception takes a message which describes the detail, so it could be reused for other types of property-relationship erros.